packages/react-server/package.json

```diff
@@ -30,10 +30,13 @@
     "@shopify/react-html": "^9.2.1",
     "@shopify/react-network": "^3.2.0",
     "@shopify/sewing-kit-koa": "^6.1.1",
+    "react-cookie": "4.0.1",
+    "@shopify/react-csrf-universal-provider": "1.0.2",
```

Comment:
All the dependency that a project may already have install should be a range instead of a specific version.

ie.
```
"@shopify/react-network": ">=3.2.0",
"@shopify/react-html": ">=8.0.0"`
"@shopify/react-csrf-universal-provider": ">=1.0.0"
...all the other ones
```